### ISSUE TITLE
[A11y] Collection Landing - Image purpose - OSULP image

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1830]
### Operating System

Windows

### Browser

Edge

### Affected page

Collection Show

### Admin user?

Yes

### Issue Description

The "Oregon State University Libraries and Press" image is coded as decorative, but I'm not sure if that's true.  It certainly tells me something about the collection I'm looking at -- that it's an OSULP collection, for instance.  The logo could go away, but the text inside the image seems meaningful to me.

### Suggested Fix

Add an alt tag to collection landing pages' images?

I don't know how these are set up, ideally there would be a logo separately from the text so that there's no concern about the accessible name of the image.  If it were just a logo next to text, the image could be decorative without any problems.

**Comment by jechols** [comment]
Also worth noting is that the image's "Libraries and Press" text disappears when in high contrast mode.  This isn't a failure if the image can be shown to be decorative, but if it's meaningful, this would be a new failure....

**Comment by jechols** [comment]
QA depends on remediation
